homeassistant/components/aladdin_connect/strings.json

```diff
@@ -0,0 +1,21 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
```

Comment:
Doesn't seem to be part of schema?

Reply:
Came from the scaffold... another amateur mistake.